pkgs/development/tools/img/default.nix

```diff
@@ -0,0 +1,58 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+, makeWrapper
+, runc
+}:
+
+buildGoModule rec {
+  pname = "img";
+  version = "0.5.11";
+
+  src = fetchFromGitHub {
+    owner = "genuinetools";
+    repo = "img";
+    rev = "v${version}";
+    sha256 = "0r5hihzp2679ki9hr3p0f085rafy2hc8kpkdhnd4m5k4iibqib08";
+  };
+
+  vendorSha256 = null;
+
+  postPatch = ''
+    substituteInPlace ./internal/unshare/unshare.c \
+      --replace "/usr/bin/newgidmap" "/run/wrappers/bin/newgidmap" \
+      --replace "/usr/bin/newuidmap" "/run/wrappers/bin/newuidmap"
```

Comment:
Shouldn't this point to something like `{package}/bin/{newgidmap,newuidmap}` instead :thinking: ?

Reply:
Like here: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/security/nsjail/default.nix#L27

Reply:
If for some reason you really need to use binaries from `/run`, I think your package will only work on NixOS.

Reply:
Thanks for your insightful input and assistance. I have parameterized the wrapper directory.